kyaml/kio/pkgio_reader_test.go

```diff
@@ -94,18 +49,18 @@ func TestLocalPackageReader_Read_empty(t *testing.T) {
 }
 
 func TestLocalPackageReader_Read_pkg(t *testing.T) {
-	s := setupDirectories(t, filepath.Join("a", "b"), filepath.Join("a", "c"))
-	defer s.clean()
-	s.writeFile(t, filepath.Join("a_test.yaml"), readFileA)
-	s.writeFile(t, filepath.Join("b_test.yaml"), readFileB)
-	s.writeFile(t, filepath.Join("c_test.yaml"), readFileC)
-	s.writeFile(t, filepath.Join("d_test.yaml"), readFileD)
+	s := SetupDirectories(t, filepath.Join("a", "b"), filepath.Join("a", "c"))
```

Comment:
We should move this to cli-utils.
It can do all of this.
https://github.com/kubernetes-sigs/kustomize/blob/master/api/filesys/filesystem.go